CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 CHANGELOG
 =========
+
+# 7.0.0 / XX-XX-2021
+## Breaking changes
+### Enable client side aggregation by default for simple metric types. See [#172][].
+By default, metrics are aggregated before they are sent. For example, instead of sending 3 times `my_metric:10|c|#tag1:value`, DogStatsD client sends `my_metric:30|c|#tag1:value`. You can disable client side aggregation by setting `ClientSideAggregation` property to `null`.
+For more details about how client side aggregation works see [#134][].
+
+Enabling client side aggregation has the benefit of reducing the network usage and also reducing the load for DogStatsD server (Core Agent).
+
+When an application sends a lot of different contexts but each context appear with a very low frequency, then enabling client side aggregation may take more memory and more CPU. A context identifies a metric name, a tag sets and a metric type. The metric `datadog.dogstatsd.client.aggregated_context` reported by DogStatsD C# client counts the number of contexts in memory used for client side aggregation. There is also the metric `datadog.dogstatsd.client.metrics_by_type` that represents the number of metrics submitted by the client before aggregation. 
+
+### Set good default values for UDS and UDP buffer sizes. See [#170][].
+This PR changes the default values for unix domain socket and UDP buffer sizes.
+In most cases, this change should work out of the box. Unlike Windows and Linux, on some operating systems like MacOS, the maximum unix domain socket buffer size is lower than `8192`. For these systems you have to set `StatsdMaxUnixDomainSocketPacketSize` to the appropriate value.
```

Comment:
> to the appropriate value
What is the appropriate value? Should it be set to the maximum buffer size of the socket for on the host os?